docs/antrea-network-policy.md

```diff
@@ -902,25 +904,40 @@ an `appliedTo` must resolve to. More information on ClusterGroups can be found i
 **serviceAccount**: This selects all the Pods which have been assigned a specific ServiceAccount.
 For more information on its usage, refer to [this section](#serviceaccount-based-selection).
 
-**ipBlock**: This selects particular IP CIDR ranges to allow as `ingress`
-"sources" or `egress` "destinations". These should be cluster-external IPs,
-since Pod IPs are ephemeral and unpredictable.
+**ipBlock**: This selects particular IP CIDR ranges to allow as `ingress` "sources"
+or `egress` "destinations". These should be cluster-external IPs, since Pod IPs are
+ephemeral and unpredictable.
+
+Starting with Antrea v2.2, Antrea-native policies now support an `except` field under
+ipBlock, which can be used to carve out specific CIDRs in the base IP range:
```

Comment:
ipBlock -> `ipBlock`

I don't think you applied Jiajun's suggestion BTW?

Reply:
I did, just forgot to paste it to the other place...